components/add_groups_to_team_modal/add_groups_to_team_modal.tsx

```diff
@@ -162,15 +179,18 @@ export default class AddGroupsToTeamModal extends React.PureComponent {
         }
     }
 
-    handleDelete = (values) => {
+    // public for tests
+    public handleDelete = (values: GroupValue[]): void => {
         this.setState({values});
     }
 
-    search = (term) => {
+    // public for tests
+    public search = (term: string): void => {
         this.props.actions.setModalSearchTerm(term);
     }
```

Comment:
functions like this can be simplified with immediate return:

```suggestion
    public search = (term: string): void => this.props.actions.setModalSearchTerm(term);
```